src/django_peerctl/static/peerctl/v2/peerctl.js

```diff
@@ -1543,14 +1540,14 @@ $peerctl.PeerSessionList = $tc.extend(
         $ctl.peerctl.port(),
         peer_session_apiobj,
         this.peer
-      ).element.val(policy_4_id);
+      ).element.val(policy_4_id).trigger("change.select2");
```

Comment:
@jason20c that same fix should be done to the policy selects in the peering list as well, assuming they would have the same issue, right?